frontend/packages/console-dynamic-plugin-sdk/src/app/core/reducers/ui.ts

```diff
@@ -0,0 +1,6 @@
+import { SDKStoreState } from '../../redux-types';
+import { getImpersonate } from './coreSelectors';
+
+export const impersonateStateToProps = (state: SDKStoreState) => {
+  return { impersonate: getImpersonate(state) };
```

Comment:
@fhlavac I'm wondering if we should put this in `coreSelectors.ts` instead of recreating `ui.ts`. What do you think?

Reply:
@florkbr I was thinking about it as well, it's probably better. Moved that